app/presenters/taxon_presenter.rb

```diff
@@ -40,7 +40,7 @@ def renders_as_grid?
   end
 
   def accordion_content
-    return [] unless renders_as_accordion?
+    raise Exception.new('Rendering type is not Accordion') unless renders_as_accordion?
```

Comment:
I agree with the change. I would probably raise a `StandardError` instead, though, as exceptions are reserved for things like interrupts etc.